pom.xml

```diff
@@ -278,6 +278,12 @@
                         <enableProcessChecker>native</enableProcessChecker>
                         <trimStackTrace>false</trimStackTrace>
                         <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                        <properties>
+                            <configurationParameters>
+                                junit.jupiter.execution.parallel.enabled = true
+                                junit.jupiter.execution.parallel.mode.default = concurrent
```

Comment:
Have you tried concurrent methods as well?

Reply:
I don’t think your addition is actually necessary. That is on by default and can be overridden to be disabled

Reply:
I have done a quick test and `junit.jupiter.execution.parallel.mode.classes.default = concurrent` has no effect if `junit.jupiter.execution.parallel.mode.default = concurrent` is already present, which it is :)